src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -666,8 +666,520 @@ app, err := newrelic.NewApplication(
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`ServerlessMode.PrimaryAppID`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`ServerlessMode.TrustedAccountKey`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`SpanEvents.Attributes.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`SpanEvents.Attributes.Exclude`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`SpanEvents.Attributes.Include`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`SpanEvents.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionEvents.Attributes.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionEvents.Attributes.Exclude`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionEvents.Attributes.Include`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionEvents.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionEvents.MaxSamplesStored`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Attributes.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Attributes.Exclude`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Attributes.Include`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Enabled`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Threshold.Duration`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`TransactionTracer.Threshold.IsApdexFailing`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Transport`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.BillingHostname`</td>
+      <td></td>
+      <td>`NEW_RELIC_UTILIZATION_BILLING_HOSTNAME`</td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectAWS`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectAzure`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectDocker`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectGCP`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectKubernetes`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.DetectPCF`</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>`Utilization.LocalRAMMIB`</td>
+      <td></td>
+      <td>`NEW_RELIC_UTILIZATION_TOTAL_RAM_MIB`</td>
+    </tr>
+    <tr>
+      <td>`Utilization.LogicalProcessors`</td>
+      <td></td>
+      <td>`NEW_RELIC_UTILIZATION_LOGICAL_PROCESSORS`</td>
+    </tr>
+ </tbody>
+</table>
+
+### Table notes: [#footnotes]
```

Comment:
we sometimes do <sup>1</sup> style for footnotes, but I don't think we have to do that, so I think this is probably fine and maybe even better.

Reply:
@zuluecho9 I am going to stick with this for now since these apply to multiple functions.